### DISCUSSION TITLE
Training on a SD20 768-trained CKPT

### DISCUSSION THREAD

**Discussion by SAC020** [TheLastBen/fast-stable-diffusion#820] [Q&A] [answered]
Hi,

If I've trained a ckpt on SD20 760, in order to re-train on top of it for a second iteration (I don't mean to resume the session, I mean to use it as the "base" ckpt instead of base SD20) do I have to tick the v2_model tick box in the model load section, or select 2.0 768 from the dropdown list? 

Or do I simply provide the path to my ckpt without any other selections?

![image](https://user-images.githubusercontent.com/50420272/205267357-88280a7a-5683-4542-97d7-110984e4efb4.png)

**Comment by TheLastBen** [comment]
There is no script to convert the V2 ckpt for now, use the session cell to load it.

**Reply by SAC020** [reply]
If I use the session cell to load it, do I have to also tick "resume session"? I'd like to use it as the baseline, not continue to train it vs SD

**Reply by TheLastBen** [reply]
if you load the session and tick the box resume_training, is exactly the same as using the ckpt as a base model.
and since the V2 ckpt can't be converted for now, you don't have a choice but to use the session cell, which more convenient since it also loads the instance and reg images.

**Comment by SAC020** [comment]
I tried skipping the "load model" section and reloading the session to resume training, and I get this

```
Loading session with no previous model, using the original model or the custom downloaded model
No model found, use the "Model Download" cell to download a model.
```

![image](https://user-images.githubusercontent.com/50420272/205430268-2ae641c3-021f-47eb-b770-5883365e8f4c.png)

**Reply by TheLastBen** [reply]
There is no saved model in that session

**Comment by SAC020** [comment, marked as answer]
I apologise but I do not understand what I should do different (hopefully not having to re-start the training)

What I did:

Step 1. Trained a model on SD20 768 by selecting "V2-768px" and ticking the "V2_model", did not fill any other fields. The training worked fine and produced a usable CKPT

The resulting folder structure is:

![image](https://user-images.githubusercontent.com/50420272/205433949-da12d781-2c0d-4090-b373-35ca3c1d78ad.png)

![image](https://user-images.githubusercontent.com/50420272/205433978-12ea17af-af99-46cc-89ee-dd6c0264fa2d.png)

Step 2. I want to resume training the model. According to instructions, I have not touched the "model download", went straight to create session, specified the same instance name, and it returns the error above

What should I do differently?

Thx

**Reply by TheLastBen** [reply]
make sure you have specified the exact name of the previous session

**Reply by SAC020** [reply]
Just retried and it worked. I can't understand what I did wrong previously, maybe added an extra space in the session name, otherwise I can't explain it. Thank you

I think I've seen a commit about being able to load V2-based CKPTs, however it did not seem to work for me. Is this feature implemented now?

**Reply by TheLastBen** [reply]
I will push a commit soon that will allow that

**Reply by SAC020** [reply]
Just to double check it's still not implemented, because I am getting this when trying to load a v2 CKPT

There are a lot of commits, I am not sure which is supposed to fix what, hence asking

![image](https://user-images.githubusercontent.com/50420272/205462249-a2ae7e19-2860-4683-b8eb-aadd53e080e5.png)

**Reply by TheLastBen** [reply]
you need to choose the right version

**Reply by SAC020** [reply]
Ok... how do I choose the right version?


![image](https://user-images.githubusercontent.com/50420272/205464034-06635463-86ba-4847-b43f-5a3ff18620f8.png)

**Reply by TheLastBen** [reply]
that model was probably trained before the recent conversion correction, so that's why it's causing the issue, but from now on, any model should work just fine

**Reply by SAC020** [reply]
ok, I will start a new model, then

**Comment by SAC020** [comment]
Now I'm getting a new error when trying to resume a SD20 768 session

![image](https://user-images.githubusercontent.com/50420272/205461886-9ad5b450-b247-4718-9746-f42123e0526b.png)

(sorry for not copy-pasting the error but for some reason it won't copy the text)

**Reply by SAC020** [reply]
Deleted the CKPT from the session folder, getting the same output
